reactive/azure.py

```diff
@@ -47,7 +47,8 @@ def no_requests():
 @when_all('apt.installed.azure-cli',
           'charm.azure.creds.set',
           'charm.azure.initial-role-update',
-          'endpoint.clients.requests-pending')
+          'endpoint.clients.requests-pending',
+          'config.changed')
```

Comment:
What was this change for? This will make it where requests are only handled if the config has changed. I don't think that's what we want.

Reply:
Ah okay, good catch. I need to think how will I deal with config changes then